compiler/parser/parser.peg

```diff
@@ -32,33 +32,33 @@ Decl
   = v:(ConstDecl / FuncDecl / OpDecl / TypeDecl) _ { return v, nil }
 
 ConstDecl
-  = "const" _ id:IdentifierName __ "=" __ expr:Expr {
+  = "const" _ id:Identifier __ "=" __ expr:Expr {
```

Comment:
Nit: While we're here, our usual convention is to use the name of destination field as the label.

```suggestion
  = "const" _ name:Identifier __ "=" __ expr:Expr {
```